src/OpenRiaServices.Server.EntityFrameworkCore/Framework/README.md

```diff
@@ -0,0 +1,79 @@
+﻿[![Stand With Ukraine](https://raw.githubusercontent.com/vshymanskyy/StandWithUkraine/main/banner2-direct.svg)](https://vshymanskyy.github.io/StandWithUkraine)
+
+
+## TERM OF USE
+
+By using this project or its source code, for any purpose and in any shape or form, you grant your **agreement** to all the following statements:
+
+- You **condemn Russia and its military aggression against Ukraine**
+- You **recognize that Russia is an occupant that unlawfully invaded a sovereign state**
+- You **support Ukraine's territorial integrity, including its claims over temporarily occupied territories of Crimea and Donbas**
+- You **do not support the Russian invasion or contribute to its propaganda'**
+
+This excludes usage by the Russian state, Russian state-owned companies, Russian education who spread propaganda instead of truth, anyone who work with the *filtration camps*, or finance the war by importing Russian oil or gas.
+
+- You allow anonymized telemetry to collected and sent during the preview releases to gather feedback about usage
+
+
+
+## Getting Started
+
+1. Ensure you have setup and configured `OpenRiaServices.Hosting.AspNetCore`
+2. Add a reference to *OpenRiaServices.Server.EntityFrameworkCore*
+    `dotnet add package OpenRiaServices.Server.EntityFrameworkCore`
+3. Add one or more domainservices. 
+Given that you have a EfCore model named `MyDbContext` with an entity `MyEntity` you add CRUD methods using something similar to below
+
+```csharp
+using using Microsoft.EntityFrameworkCore;
+using OpenRiaServices.Server.EntityFrameworkCore;
+
+[EnableClientAccess]
+public class MyDomainService : DbDomainService<MyDbContext>
+{
+    /* Example of CUD methods */

```

Comment:
Should I add ctor taking in MyDbContext ? I think it is safe to say that it is best practices, even if it makes the example slightly longer